activecampaign.php

```diff
@@ -98,6 +98,8 @@ function activecampaign_plugin_options() {
 				$account = $ac->api("account/view");
 				$domain = (isset($account->cname) && $account->cname) ? $account->cname : $account->account;
 				$instance["account"] = $domain;
+				// The ActiveCampaign hosted domain (something.activehosted.com).
+				$instance["account_hosted"] = $account->account;
```

Comment:
Updated now. Since we're now using `$protocol` in two different spots, I declared it further up.